source/tasks/CreateOctopusRelease/CreateOctopusReleaseV6/createRelease.ts

```diff
@@ -10,7 +10,7 @@ export async function createReleaseFromInputs(client: Client, command: CreateRel
         const repository = new ReleaseRepository(client, command.spaceName);
         const response = await repository.create(command);
 
-        client.info(`🎉 Release ${response.ReleaseVersion} created successfully!`);
+        client.info(`🎉 Release ${response.ReleaseVersion} is ready for deployment!`);
```

Comment:
If the call was successful and `IgnoreIfAlreadyExists` is false, then we know a new one was created successfully, if it is true then we don't know what happened
```suggestion
if (command.IgnoreIfAlreadyExists) {
    client.info(`🎉 Release ${response.ReleaseVersion} is ready for deployment!`);
} else {
    client.info(`🎉 Release ${response.ReleaseVersion} created successfully!`);
}
```